src/plugins/misc/commands/jumbo.command.ts

```diff
@@ -0,0 +1,52 @@
+import {ChatInputCommandInteraction, inlineCode, PermissionFlagsBits} from "discord.js";
+import {SlashCommandBuilder, SlashCommandScope} from "../../../builders/SlashCommandBuilder";
+import {useChatCommand} from "../../../hooks/useChatCommand";
+import axios from 'axios';
+
+const builder = new SlashCommandBuilder()
+    .setName("jumbo")
+    .setDescription("Make G'day blow up an image.")
+    .addStringOption(option => option.setName("emoji").setDescription("Emoji to jumbo size.").setRequired(true))
+    .setDefaultMemberPermissions(PermissionFlagsBits.ManageMessages)
```

Comment:
Probably don't need to set permissions this high

Reply:
I believe this was left over from say and it seemed like only staff used it, wasn't sure if you wanted to keep it to staff or not.

Went ahead and removed the flag in general.